app/react/Uploads/components/specs/ImportPanel.spec.js

```diff
@@ -56,5 +55,17 @@ describe('ImportPanel', () => {
         expect(component).toMatchSnapshot();
       });
     });
+    describe('error', () => {
+      describe('when there is an error', () => {
+        it('should render an error message', () => {
+          props.importStart = false;
+          props.importProgress = 0;
+          props.uploadProgress = 0;
+          render();
+          component.setState({ showError: true });
+          expect(component).toMatchSnapshot();
```

Comment:
This test is pretty basic and is using the undesired snapshot comparison, also, it's pretty similar to the existing at `ImportProgress.spec.js`

Reply:
Removed